tests/testthat/test-plumber.R

```diff
@@ -2,7 +2,10 @@ context("Plumber")
 
 exec_endpoint <- function(pr, ep_pos, subset = 1) {
   # This is a poor setup of `req` and `res`. But it works for testing purposes
-  pr$endpoints[[subset]][[ep_pos]]$exec(make_req(), PlumberResponse$new())
+  pr$endpoints[[subset]][[ep_pos]]$exec(
+    as.list(make_req()),
+    PlumberResponse$new()
+  )
```

Comment:
Reverting to surface error

```suggestion
  pr$endpoints[[subset]][[ep_pos]]$exec(make_req(), PlumberResponse$new())
```